docs/src/main/sphinx/client/jdbc.md

```diff
@@ -248,4 +248,8 @@ may not be specified using both methods.
         treated as underscores. You can use this as a workaround for
         applications that do not escape schema or table names when passing them
         to ``DatabaseMetaData`` methods as schema or table name patterns.
+   * - ``timezone``
+     - Sets the time zone for the session using the `time zone name
+       <https://wikipedia.org/wiki/List_of_tz_database_time_zones>`_. Defaults
+       to the timezone set on your workstation.
```

Comment:
Instead of workstation, this should say "the timezone of the JVM running the JDBC driver", since the JDBC driver can be used in many contexts.